server/command.go

```diff
@@ -142,21 +150,29 @@ func (p *Plugin) runStartCommand(args *model.CommandArgs, user *model.User, topi
 		}
 		return authErr.Message, authErr.Err
 	}
-
-	client, _, err := p.getActiveClient(user)
-	if err != nil {
-		p.API.LogWarn("Error creating the client", "err", err)
-		return "Error creating the client.", nil
+	var meetingID int = -1
+	var createMeetingErr error = nil
+
+	if userPMISettingPref, getUserPMISettingErr := p.getPMISettingData(user.Id); getUserPMISettingErr == nil {
```

Comment:
what if getUserPMISettingErr is not nil you are not checking this anywhere and I think this would have been better if you check for not nil error rather than nil